### PR TITLE
:memo: docs: organize config.mts

### PR DESCRIPTION
**Pull request by Sabnock01**
* Organizes the actions in alphabetical order ([viem.sh](https://viem.sh/docs/getting-started.html) also uses alphabetical ordering of actions) 
* Creates and adds missing pages to index (will implement those once this is merged; plenty of empty pages already exist)
* Fixes a typo in the sidebar for `simulateWithdrawERC20`

Let me know if we prefer another name to the `General` category under `Utilities` for functions like `resolveL1OpStackContractAddress`.

### CODE REVIEW

**Comment by Sabnock01** [comment]
Decided `writeOpStackL1` and `resolveL1OpStackContractAddress`should be included in this PR.

**Comment by wilsoncusack** [comment]
Thanks for this! I was unsure if we wanted to expose `writeOpStackL1`, I think it is not currently exported. Got some feedback the name was confusing and there was some debate if this function is worth having (I currently think it probably is). Any thoughts @Sabnock01 ?

File: site/.vitepress/config.mts [hunk 1]
```diff
@@ -171,6 +179,15 @@ export default defineConfig({
               },
             ],
           },
+          {
+            text: 'General',
+            items: [
+              {
+                text: 'resolveL1OpStackContractAddress',
```
Comment (wilsoncusack) [inline site/.vitepress/config.mts:186]: I think this is internal and we don't need to export

**Review by wilsoncusack — APPROVED** [review]

**Comment by Sabnock01** [comment]
> > > Thanks for this! I was unsure if we wanted to expose `writeOpStackL1`, I think it is not currently exported. Got some feedback the name was confusing and there was some debate if this function is worth having (I currently think it probably is). Any thoughts @Sabnock01 ?
> > 
> > 
> > I am fine with exporting `writeOpStackL1` and leaving `resolveL1OpStackContractAddress` ad internal.
> 
> ok, let's do that then. I also wonder if writeOpStackL1Contract might be a more clear naming.

I think it is. Want to rename to both `writeOpStackL1Contract` and `writeOpStackL2Contract`?

**Comment by wilsoncusack** [comment]
@Sabnock01 I am open to this but let's make it a different PR. Maybe this one can just be the reorg and we can punt on writeOpStackL1 and plan to not have docs or export `resolve...`

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**